packages/aws-cdk/lib/init-templates/jsii/typescript/package.template.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "%name%",
+  "version": "0.1.0",
+  "description": "A sample CDK app",
+  "keywords": [
+    "cdk",
+    "sns",
+    "sqs",
+    "jsii"
+  ],
+  "author": {
+    "name": "[YOUR NAME HERE]",
+    "email": "[YOUR EMAIL HERE]",
+    "url": "[YOUR WEBSITE HERE]"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "[GIT URL HERE]",
+    "directory": "lib/"
+  },
```

```suggestion
  "repository": {
    "type": "git",
    "url": "[GIT URL HERE]"
  },
```

The `directory` value is a lie in the context of this template... the package is rooted at the root of the generated repository, not within `lib`.